mlflow/tracking/fluent.py

```diff
@@ -286,6 +303,34 @@ def log_artifacts(local_dir, artifact_path=None):
     MlflowClient().log_artifacts(run_id, local_dir, artifact_path)
 
 
+def list_run_infos(experiment_id, run_view_type=ViewType.ACTIVE_ONLY):
+    """:return: List of :py:class:`mlflow.entities.RunInfo`"""
+    return MlflowClient().list_run_infos(experiment_id, run_view_type)
+
+
+def list_experiments(view_type=None):
```

Comment:
I also wonder if we could hold off on exposing `list_experiments` via the fluent API for now, in part because as an unpaginated API it can put a lot of load on the server & we should encourage users to use `mlflow.get_experiment` and `mlflow.get_experiment_by_name` instead to query for specific experiments.